fastlane/lib/fastlane/helper/sh_helper.rb

```diff
@@ -54,7 +54,8 @@ def self.sh_control_output(*command, print_command: true, print_command_output:
         Open3.popen2e(*command) do |stdin, io, thread|
           io.sync = true
           io.each do |line|
-            UI.command_output(line.strip) if print_command_output
+            # make sure string that will be stripped has valid utf8, otherwise will replace with invalid character.
```

Comment:
```suggestion
            # Ensure the string being stripped is in valid UTF-8 format; otherwise, it will be replaced with an invalid character.
```

Reply:
thanks